packages/fetchai/agents/generic_buyer/README.md

```diff
@@ -0,0 +1,12 @@
+# Generic Buyer
+
+A generic agent for buying data.
+
+## Description
+
+This agent uses its primary skill, the `generic_buyer` skill, to  find an agent selling data on the `SOEF` service. Once found, it requests specific data, negotiates the price using the `fipa` protocol, and if an agreement is reached, pays the proposed amount and receives the data.
+
+## Links
```

Comment:
These links end up being outdated very fast. We can add them now but should be mindful of that.

Also, when there are no links please don't add a useless heading (like in some of the connections)

Reply:
Agree. That was for template consistency.